### DISCUSSION TITLE
WalletConnect Project ID in "draft" status

### DISCUSSION THREAD

**Discussion by GregEPersonal** [WalletConnect/walletconnect-monorepo#2882] [Developer Support] [answered]
Hello, I have a dapp that allows users to connect their wallet and sign messages for basic verification of ownership. We're using RainbowKit, which leverages WalletConnect on the backend. 

I've gotten a project ID, but it's listed in draft form. Will that work?

Second question - I'd like to publish, but we're a website only - there's no "download link". The Wallet Connect project profile says "Add your project download links. At least 1 is required." What should we do there?

Thanks.

**Comment by JeroldG** [comment]
Could you share your project ID, 

I think you can do this by selecting the type field to dapp, this will remove the need to share download links.

**Comment by chadyj** [comment, marked as answer]
> I've gotten a project ID, but it's listed in draft form. Will that work?

@GregEPersonal Yes, you can use the projectid immediately.
The "draft" status refers to your [explorer listing publication status](https://walletconnect.com/explorer).

**Reply by GregEPersonal** [reply]
Thank you!
